modules/nf-core/bbmap/bbmerge/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process BBMAP_BBMERGE"
+    script "../main.nf"
+    process "BBMAP_BBMERGE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bbmap"
+    tag "bbmap/bbmerge"
+
+    test("sarscov2_illumina_paired - fastq_gz") {
+
+
+        when {
+            process {
+                """
+
+                input[0] = Channel.of([
+                    [ id:'test', single_end:false ], // meta map
+                    [file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true)]
+                    ])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
This might fail later, consider testing what you find in the files but do test at least the versions match:
```suggestion
                { assert snapshot(process.out.version).match("version") }
```

Some examples:
```
# checking output vcf files
{ assert path(process.out.candidate_small_indels_vcf.get(0).get(1)).linesGzip.contains("##fileformat=VCFv4.1") }

# checking output files with same headers
{ assert snapshot(file(process.out.stats.get(0).get(1)).readLines()[0..5]).match() }
```